webview-codicons-sample/media/styles.css

```diff
@@ -56,5 +56,44 @@ h1 {
 
 #icons .icon .codicon {
 	font-size: 32px;
-	padding-bottom: 16px;
+}
+
+/* Codicon mouse events demo */
+
+div.iconBar .codicon {
+	font-size: 60px;
+	cursor: pointer;
+}
+
+div.iconBar .disabled {
+	opacity: 0.3;
+	cursor: not-allowed;
+}
+
+/* Codicons in buttons demo */
+
+div.iconButtonBar .iconButton .codicon {
+	font-size: 40px;
+	padding: 3px;
+}
+
+div.styledIcon .codicon {
+	font-size: 50px;
+	color: green;
+	padding: 3px;
+}
+
+/* Rotating the 'loading' Codicon to indicate _progress_. */
+
+.rotate {
+	animation: rotation 2s infinite linear;
+}
+
+@keyframes rotation {
+from {
+	transform: rotate(0deg);
+}
+to {
+	transform: rotate(359deg);
+}
```

Comment:
I'd try using the same styles from out regular spinners here https://github.com/microsoft/vscode/blob/master/src/vs/base/browser/ui/codicons/codicon/codicon-animations.css

Reply:
I cannot see how to use the same css definition, so I copied it, because it is indeed a better implementation.